builder/docker/ecr_login.go

```diff
@@ -33,14 +33,12 @@ type AwsAccessConfig struct {
 	// probably don't need it. This will also be read from the AWS_SESSION_TOKEN
 	// environmental variable.
 	Token string `mapstructure:"aws_token" required:"false"`
-	// The AWS shared credentials profile used to
-	// communicate with AWS. Learn how to set
-	// this.
+	// The AWS shared credentials profile used to communicate with AWS.
```

Comment:
If we're removing the "Learn how to set this.", we should probably remove them from the Access/Secret Key attributes too.

Good job on removing those otherwise, I think it's good to be more concise on those comments!